program1.py

```diff
@@ -0,0 +1,77 @@
+"""
+this is first program to counte the number of vowels, words, sentences, and punctuation marks.
+"""
+
+from importlib.resources import contents
+from string import punctuation
+
+
+
+def count_vowels(test_Sentence):
+    number_of_vowels=0
+    """
+    this function is used to count vowels
+    Parameters:
+                    test_sentence (str): stores sentence set by user or data read from a file
+
+            Returns:
+                   number_of_vowels (int) : returns count of vowels in sentence
+    
+    """
+    for element in test_Sentence:
+        if element=="a" or element=="e" or element=="i" or element=="o" or element=="u":
+            number_of_vowels=number_of_vowels+1
+    return number_of_vowels
+
+def count_words(test_Sentence):
+    
+    """
+    this function is used to count total number of words 
+    Parameters:
+                    test_sentence (str): stores sentence set by user or data read from a file
+    
+    """
+    number_ofword=test_Sentence.split()
+    print(len(number_ofword))
+
+
+def count_puntuation_marks(sentence):
+    number_of_punctuation_marks=0
```

Comment:
All the local variables should be initialized after the doc string. Please run a `pylint` before creating a PR.